index.html

```diff
@@ -4,70 +4,48 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Graphene UI</title>
   </head>
   <body>
     <main>
       <nav class="navbar">
         <div class="header-logo">
-          <button id="btn-ham-burger"><i class="far fa-bars"></i></button>
-          <a class="head-title" href="">Graphene UI</a>
+          <a class="head-title" href="/">Graphene UI</a>
         </div>
         <div class="hdr-icons">
-          <a class="btn-social-icons" target=”_blank” href="https://github.com/fahd23/Graphene-UI">
+          <a href="/documentations/docs/index.html" class="p-s"
+            >Documentation</a
+          >
+          <a
+            class="btn-social-icons"
+            target="”_blank”"
```

Comment:
check if this is working

Reply:
Yeah it is, Anyway i will remove it, Thanks for pointing out